src/server/src/main/java/io/cassandrareaper/storage/CassandraStorage.java

```diff
@@ -570,14 +570,16 @@ private void prepareMetricStatements() {
                 "INSERT INTO reaper_db.running_repairs(repair_id, node)"
                     + "values (?, ?) IF NOT EXISTS")
             .setSerialConsistencyLevel(ConsistencyLevel.SERIAL)
-            .setIdempotent(true);
+            .setConsistencyLevel(ConsistencyLevel.QUORUM)
+            .setIdempotent(false);
     setRunningRepairsPrepStmt
       = session
         .prepare(
             "UPDATE reaper_db.running_repairs USING TTL ?"
             + " SET reaper_instance_host = ?, reaper_instance_id = ?, segment_id = ?"
             + " WHERE repair_id = ? AND node = ? IF reaper_instance_id = ?")
         .setSerialConsistencyLevel(ConsistencyLevel.SERIAL)
+        .setConsistencyLevel(ConsistencyLevel.QUORUM)
```

Comment:
A write at QUORUM would not fail with a single node down, even in a multi DC cluster.
We can't use LOCAL CLs here because two Reaper instances in different DCs could take the lead on the same nodes/segments and cause inconsistencies and weird behaviors.
This operation has to involve all DCs.
I know these would fail in case of a topology change, which is fine in the context of repairs (token assignements would change and some segments would be invalid anyway).

Reply:
> A write at QUORUM would not fail with a single node down, even in a multi DC cluster.

Yes it will a known percentage of times.
Writes to remote DCs are "forwarded" to a coordinator in the remote DC. That is never retried. If a node is unresponsive in the remote DC then 1/NDC (where NDC is number of nodes in that remote DC) multi-DC QUORUM write requests will fail as they attempt to use that unresponsive node as a remote coordinator.

If a node is unresponsive then all multi-DC QUORUM writes fail 1/NDC % of the time.

I know LOCAL_QUORUM is not an option, but QUORUM writes in a multi-dc cluster remain an anti-pattern and we need to expect and handle failures.